src/grammar.rs

```diff
@@ -216,6 +237,27 @@ impl<Pat> RuleWithNamedFields<Pat> {
         self.rule = Rc::new(Rule::RepeatMore(self.rule, sep.map(|sep| sep.rule)));
         self
     }
+    pub fn repeat_more_trailing(mut self, sep: Self) -> Self {
+        self.fields = self
+            .fields
+            .into_iter()
+            .map(|(name, paths)| {
+                (
+                    name,
+                    paths
+                        .into_iter()
+                        .map(|mut path| {
+                            path.insert(0, 0);
+                            path
+                        })
+                        .collect(),
+                )
+            })
+            .collect();
```

Comment:
From the structure of things ISTM that everything up to here is identical between `repeat_{many|more}{_trailing}?` -- refactor?